content/it/style-guide/_index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Guida di Stile
+toc_hide: true
+menu:
+  main:
+    weight: 10
+    pre: <i class='fas fa-ruler-horizontal'></i>
+---
+
+Questa guida ti aiuterà a comprendere la target audience del Glossario, la struttura delle definizioni, il livello di dettaglio richiesto e come mantenere uno stile coerente.
+
+Il Glossario Cloud Native rispetta la [style guide di base](https://github.com/cncf/foundation/blob/master/style-guide.md) del repository CNCF, ma si è posto regole aggiuntive e specifiche:
+
+1. Usa un linguaggio semplice, accessibile e che eviti tecnicismi spinti e buzzwords
+2. Evita un linguaggio eccessivamente colloquiale
+3. Usa un linguaggio concreto ed espressioni letterali
+4. Fai attenzione agli accenti, alla correlazione dei tempi verbali e alla punteggiatura
+5. Prediligi la forma attiva
+6. Utilizza preferibilmente la forma affermativa
+7. Non eccedere con le parentesi
+8. Non esagerare
+9. Evita le ripetizioni
+10. Sii concisə
+11. Usa pure termini e diciture in inglese se sono largamente utilizzati in ambito tecnico, ma aggiungi comunque una traduzione italiana per chiarire.
+
+Consigliamo di dare un'occhiata a questi [set di regole grammaticali](https://grammatica-italiana.dossier.net/grammatica-italiana-17.htm) e alle [40 regole di scrittura di Umberto Eco per scrivere bene in italiano](https://bologna.unicusano.it/universita/scrivere-correttamente-in-italiano/). Possono essere di aiuto.
+
+
+## Audience
+
+Il Glossario è scritto sia per un'audience tecnica sia per un'audience NON-tecnica. Per questa ragione assicurati che le definizioni siano espresse in termini semplici e che non diano per scontata una conoscenza tecnica pregressa. Lo spieghiamo meglio nel paragrafo *Definizioni*.
+
+## Template Definizioni
+
+Ogni termine del glossario è contenuto in un file di markdown e segue questo schema: 
+
+```md
+---
+title: 
+status: 
+category: 
+---
+
+## Cos'è
+
+Una breve spiegazione della tecnologia o del concetto. 
+
+## Quali problematiche risolve
+
+Qualche riga per inquadrare il problema che viene affrontato. 
+
+## In cosa aiuta
```

Comment:
credo sia più indicato
```suggestion
## In che modo aiuta
```
`In cosa aiuta` mi pare più siile al titolo precedente